README.md

```diff
@@ -7,6 +7,10 @@
 
 A Mac app for designers to specify colors, fonts, and metrics in a standardized format for developers to consume.
 
+Download latest build here:
+[https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358](https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358)  
```

Comment:
How about
`[Download the latest build from HockeyApp](https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358)`?